great_expectations/core/batch_manager.py

```diff
@@ -0,0 +1,166 @@
+import logging
+from collections import OrderedDict
+from typing import Dict, List, Optional
+
+from great_expectations.core.batch import (
+    Batch,
+    BatchData,
+    BatchDefinition,
+    BatchMarkers,
+)
+from great_expectations.core.id_dict import BatchSpec
+
+logger = logging.getLogger(__name__)
+logging.captureWarnings(True)
+
+
+class BatchManager:
+    def __init__(
+        self,
+        execution_engine: "ExecutionEngine",  # noqa: F821
```

Comment:
I think you can get around this quotation method by doing: 

```
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from great_expectations.execution_engine import ExecutionEngine
```

Reply:
@NathanFarmer Trying... :)

Reply:
@alexsherstinsky this comment from @Kilo59 will allow you to remove the quotations